specs/engine.spec/src/main/scripts/io/aklivity/zilla/specs/engine/schema/engine.schema.json

```diff
@@ -303,6 +303,112 @@
                 }
             ]
         },
+        "options":
+        {
+            "title": "Options",
+            "type": "object",
+            "properties":
+            {
+                "binding":
+                {
+                    "title": "Binding",
+                    "type": "object",
+                    "properties":
+                    {
+                        "http":
```

Comment:
We can't have http-specific stuff in the engine schema.

Instead this should be contributed by the http schema patch.

Reply:
that's what I was not sure when we were discussing and agreed on this `suggest we choose something like $defs/options/binding/http and then refer to it from http binding and openapi binding`

Does it mean we keep the path and only contribute the content as path.

Reply:
Right, we need `$defs/options/binding` to exist in engine schema, then contribute `http` under that via http schema patch.

For example, when we make any change to the http schema (whether it is in authorization or not), the PR should only have files modified in the http spec project.